.travis.yml

```diff
@@ -90,4 +90,8 @@ install:
 
 script:
 
+  - >
+    echo $(black --version);
+    black --check $(INSTALL_DIR);
```

Comment:
Have not replicated [Iris' line removing .gitignore](https://github.com/SciTools/iris/blob/d16f6767d4d775e2afd691de9c6514782ba2a90d/.travis.yml#L111) - testing has shown that Black is successfully skipping .gitignore automatically.

Reply:
I think Iris also has the line `  - export INSTALL_DIR=$(pwd)` within the script section rather than the install section. I don't know if this is relevant or not.

Reply:
It's present here too, but further up - [line 10](https://github.com/SciTools-incubator/iris-ugrid/blob/d79a304ebd4de353e37f2d3701fada40a3e02609/.travis.yml#L10)

Reply:
The current travis run seems to give "No Path provided. Nothing to do 😴" despite passing. I wonder if `- export INSTALL_DIR=$(pwd)` is only applying to the contex of the `install:` block of code and should be moved down to below line 91 where the `script:` section begins.

Reply:
It seemed safer to declare it a second time. As you've alluded to, it seems that environment variables are independent between the code blocks.

Reply:
Well, "No Python files are present to be formatted. Nothing to do 😴" seems like an improvement at least.

Reply:
From local testing, I'm expecting:
All done! ✨ 🍰 ✨

Reply:
The mystery now is why I get 3 passes locally, but on Travis it doesn't like `__init__.py`...

Reply:
From what I can tell, the only thing that needs changing now is a newline at the end of the `__init__.py` file, and I suppose it's good to know it's picking that up. I'd say this is good to go once that's done.